Multiple idps (#205)
mulesoft/docs-api-manager: modules/ROOT/pages/import-api-task.adoc

You can develop an API for and export it from API Manager for future use or for sharing. The exported API can be imported using this procedure. To perform this task, you need to have a ZIP file on your file system containing an API object exported from API Manager.

. After logging into Anypoint Platform, click API Manager.

> . In API Administration, select *Manager API > Import API from the Deskto*p.

Reviewer: ```suggestion
. In API Administration, select *Manager API > Import API from the Desktop*.
```

Reviewer: Because of the misplaced format symbol *, the following steps appear here in bold. Please verify that this does not carry over in the production version.

Author: Done! Thanks!

> . Browse to and select the file ZIP file that contains the exported API instance to import.

Reviewer: ```suggestion
. Browse to and select the ZIP file that contains the exported API instance to import.
```

Author: Done! Thanks!

> . Click Import.

Reviewer: ```suggestion
. Click *Import*.
```

Author: Done! Thanks!

> . If external IdPs are configured for your business organization or environment, select the IdP that you want to use to authenticate access to this API from registered client applications.

Reviewer 2: some nits:
* Let's use Client Providers instead of IdPs
* Client Providers are not configurable for business groups
* Authenticate -> Authorize

> . If external IdPs are configured for your business organization or environment, select the IdP that you want to use to authenticate access to this API from registered client applications.

Reviewer: ```suggestion
. If external IdPs are configured for your business organization or environment, select the IdP that you want to use to authenticate access to this API from registered client applications.
```

Author: No change visible.

> == See Also

Reviewer: ```suggestion
== See Also
```

Author: Done! Thanks!

> xref:export-api-latest-task.adoc[Exporting an API instance on API Manager]

Reviewer: ```suggestion
xref:export-api-latest-task.adoc[Exporting an API Instance on API Manager]
```

Author: Done! Thanks!
